pkg/cmd/server/start/start_master.go

```diff
@@ -306,11 +306,13 @@ func StartMaster(openshiftMasterConfig *configapi.MasterConfig) error {
 
 	unprotectedInstallers := []origin.APIInstaller{}
 
-	authConfig, err := origin.BuildAuthConfig(*openshiftMasterConfig)
-	if err != nil {
-		return err
+	if openshiftMasterConfig.OAuthConfig != nil {
```

Comment:
"Support" is such a strong term... this lets you start up and use cert auth only without panicking on invalid pointer access. I didn't check much more than that